quick-start/src/main/java/com/marklogic/hub/service/FileSystemWatcherService.java

```diff
@@ -131,28 +131,31 @@ private void registerAll(final Path start) throws IOException {
             public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs)
                 throws IOException
             {
+                if (dir.endsWith("REST")) {

```

Comment:
Not clear why REST is skipped from this snippet. A quick comment would be useful here.